src/node_options.cc

```diff
@@ -402,6 +402,11 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::force_async_hooks_checks,
             kAllowedInEnvironment,
             true);
+  AddOption("--addons",
```

Comment:
Was this supposed to be `--no-addons`?

Reply:
We generally use the `no-`-less version here since https://github.com/nodejs/node/pull/39023

Reply:
Yep, it's negated and the parser does that actually. See https://github.com/nodejs/node/blob/1bbe66f432591aea83555d27dd76c55fea040a0d/src/node_options-inl.h#L337